test/functional/p2p_disconnect_ban.py

```diff
@@ -77,6 +77,7 @@ def run_test(self):
         self.nodes[1].setmocktime(old_time)
         self.nodes[1].setban("127.0.0.0/32", "add")
         self.nodes[1].setban("127.0.0.0/24", "add")
+        self.nodes[1].setban("pg6mmjiyjmcrsslvykfwnntlaru7p5svn6y2ymmju6nubxndf4pscryd.onion", "add")
```

Comment:
nit:  The line just after this has an index that isn't adjusted, but still passes.
```python
assert_equal("192.168.0.1/32", listBeforeShutdown[2]['address'])
```
At first glance, this might be due to how the address types are organized in the list returned by `listbanned()` but if this organization changes then the test could fail in the future.  Recommend this be updated to increase robustness.  For example:
```python
addrs_before_shutdown = [a['address'] for a in self.nodes[1].listbanned()]
assert "192.168.0.1/32" in addrs_before_shutdown
```
Author's call as to whether this is part of this PR or a new/future one.  There are other parts of test that depend on specific ordering in the list (this isn't the only spot).

Reply:
I'll leave it as it for now. Perhaps we could change it if/when when the behaviour of `listbanned` changes.